sample-apps/discount-functions-sample-app/web/frontend/components/DiscountCreatePage.jsx

```diff
@@ -56,19 +83,62 @@ export default function DiscountCreatePage({
       <Layout>
         {errorMarkup}
         <Layout.Section>
+          <MethodCard
+            title="Create discount"
+            discountClass={discountClass}
+            discountTitle={{
+              value: title,
+              onChange: setTitle,
+            }}
+            discountCode={{
+              value: code,
+              onChange: setCode
+            }}
+            discountMethod={{
+              value: method,
+              onChange: setMethod
+            }}
+          />
           <Card>
-            <Card.Section>
-              <TextField
-                value={title}
-                onChange={setTitle}
-                label="Discount title"
-                autoComplete="on"
-              />
-            </Card.Section>
             <Card.Section>
               {renderConfigurationForm(configuration, setConfiguration)}
             </Card.Section>
           </Card>
+          {method === DiscountMethod.Code && (
+            <UsageLimitsCard
+              totalUsageLimit={{
+                value: usageLimit,
+                onChange: (value) => setUsageLimit(parseInt(value)),
```

Comment:
(we talked about this in slack but leaving it here too) This breaks the input, it expects a string. Suggest changing to `onChange: setUsageLimit` and casting to an int in the submit mutation